### DISCUSSION TITLE
SNES PAL Internal Mod - where to wire the 3.6k resistors :/

### DISCUSSION THREAD

**Discussion by elboffor** [darthcloud/BlueRetro#653] [Q&A] [answered]
Hi,
I've tried a few times to build an internal mod for my SNES (PAL region) but I can't for the life of me get any button presses to register on the console. any support would be appreciated. I have the firmware flashed and pads can connect etc.

I'm fairly new to this kind of thing so please forgive my lack of understanding. 
I think the issue is with the placement of the 3.6k resistors, from what I have read I believe they need to be connected to the 5v rail and then to the various pins that go between the SNES and the ESP, is that right or have I totally got it wrong?
example j1 pin 2 to both 3.6k resistor and the ESP, and the other end of the resistor goes to the 5v rail?

My plan is to fit this internally, print out a mounting bracket (will share the STL files here after I design it), will also mount a dip switch behind the controller port to act as a kill switch for when a pad is inserted and also blue LEDs to illuminate the ports when they are in BT mode. but first hurdle of course is to get the thing working :/

**Comment by elboffor** [comment, marked as answer]
I've done some research and my assumption was correct. Turned it I had a dead esp board

**Reply by conkydong** [reply]
So where did you wire the resistors?
